docs/product-user-guide/src/main/asciidoc/guided-decision-tables-create-proc.adoc

```diff
@@ -7,21 +7,22 @@ You can use guided decision tables to define rule attributes, meta-data, conditi
 . Go to *Menu* -> *Design* -> *Projects* and select your project.
 . Click *Create New Asset* -> *Guided Decision Table*.
 . Enter an informative *Guided Decision Table* name and select the appropriate *Package*.
-. Select *Use Wizard* to finish setting up the table in the Wizard, or leave this option unselected to finish creating the table and specify remaining configurations in the guided decision table editor.
-+
-The Wizard helps you define your imports, facts, patterns, and columns, but not the rows. Rows are added in the guided decision table editor after initial table setup.
-+
+. Select *Use Wizard* to finish setting up the table in the Wizard, or leave this option unselected to finish creating the table and specify remaining configurations in the guided decision table editor. 
```

Comment:
in the Wizard.. w should be l/c. I know this isn't new text, but while you are editing...

Reply:
I'll fix that but need to understand how to work pull requests if I edit stuff. If I rescind this pull request we lose the discussion?